src/dev/code_coverage/ingest_coverage/transforms.js

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { left, right } from './either';
+import { always, id, noop } from './utils';
+
+const maybeTotal = x => (x === 'total' ? left(x) : right(x));
+
+export const trimLeftFrom = (text, x) => x.replace(new RegExp(`(?:.*)(${text}.*$)`, 'gm'), '$1');
```

Comment:
Is RegEx really necessary here?

`export const trimLeftFrom = (text, x) => x.substr(x.indexOf(text)) `

Reply:
We dont like Regex?

Reply:
The added cost and overhead of RegEx doesn't seem worth it when a simple substring can be used.

Reply:
Sure!